index.html

```diff
@@ -352,6 +352,20 @@ <h2 id="add-website">Adding ESP Web Tools to your website</h2>
         ESP Web Tools can also be integrated in your projects by installing it
         <a href="https://www.npmjs.com/package/esp-web-tools">via NPM</a>.
       </p>
+      <h3 id="preparing-firmware">Preparing your firmware</h3>
+      <p>
+        ESP32 firmware is usually split into 4 different files. When these files
+        are installed using `esptool`, it will patch certain bytes to match the
+        target device. Because ESP Web Tools is not able to do this, you will need
+        to use `esptool` to create a single binary file of your firmware and point
```

Comment:
```suggestion
        are installed using <code>esptool</code>, it will patch certain bytes to match the
        target device. Because ESP Web Tools is not able to do this, you will need
        to use <code>esptool</code> to create a single binary file of your firmware and point
```